source/server/config.cpp

```diff
@@ -69,483 +69,747 @@ static unsigned int s_heartbeat_retry_count(5);
+void setRulesFile(const std::string &file)
+{
+   s_rulesfile = file;
+}
 
-    void setMaxVehicles(unsigned int num) { s_max_vehicles = num; }
+void setMaxVehicles(unsigned int num)
+{
+   s_max_vehicles = num;
+}
 
-    void setOwner(const std::string &owner) { s_owner = owner; }
+void setOwner(const std::string &owner)
+{
+   s_owner = owner;
+}
 
-    void setForeground(bool value) { s_foreground = value; }
+void setForeground(bool value)
+{
+   s_foreground = value;
+}
 
-    void setWebsite(const std::string &website) { s_website = website; }
+void setWebsite(const std::string &website)
+{
+   s_website = website;
+}
 
-    void setIRC(const std::string &irc) { s_irc = irc; }
+void setIRC(const std::string &irc)
+{
+   s_irc = irc;
+}
 
-    void setVoIP(const std::string &voip) { s_voip = voip; }
+void setVoIP(const std::string &voip)
+{
+   s_voip = voip;
+}
 
-    void setHeartbeatIntervalSec(unsigned sec) {
-        s_heartbeat_interval_sec = sec;
-        Logger::Log(LOG_VERBOSE, "Hearbeat interval is %d seconds", sec);
-    }
+void setHeartbeatIntervalSec(unsigned sec)
+{
+   s_heartbeat_interval_sec = sec;
+   Logger::Log(LOG_VERBOSE, "Hearbeat interval is %d seconds", sec);
+}
 
-    bool setServerMode(ServerType mode) {
-        s_server_mode = mode;
-        return true;
-    }
+bool setServerMode(ServerType mode)
+{
+   s_server_mode = mode;
+   return true;
+}
 
-    void setResourceDir(std::string dir) {
-        if (dir.length() > 0 && dir.substr(dir.length() - 1) != "/")
-            dir += "/";
-        s_resourcedir = dir;
-    }
+void setResourceDir(std::string dir)
+{
+   if (dir.length() > 0 && dir.substr(dir.length() - 1) != "/")
+      dir += "/";
+   s_resourcedir = dir;
+}
 
-    inline void SetConfServerMode(std::string const &val) {
-        if (val.compare("inet") == 0)
-            setServerMode(SERVER_INET);
-        else
-            setServerMode(SERVER_LAN);
-    }
+inline void SetConfServerMode(std::string const &val)
+{
+   if (val.compare("inet") == 0)
+      setServerMode(SERVER_INET);
+   else
+      setServerMode(SERVER_LAN);
+}
 
 // Cloned from RudeConfig (http://rudeserver.com/config/) we used before. #compatibility
-    inline bool RudeStrToBool(const char *val) {
-        switch (val[0]) {
-            case 't':
-            case 'T':
-            case 'y':
-            case 'Y':
-            case '1':
-                return true;
-            case 'o':
-            case 'O':
-                return (val[1] == 'n' || val[1] == 'N');
-            default:
-                return false;
-        }
-    }
-
-#define VAL_STR(_STR_) (Str::SanitizeUtf8(_STR_))
-#define VAL_INT(_STR_) (atoi(value)) // Cloned from RudeConfig we used before. #compatibility
-#define VAL_BOOL(_STR_)(RudeStrToBool(value))
-
-    void ProcessConfigEntry(const char *key, const char *value) {
-        if (strcmp(key, "baseconfig") == 0) { LoadConfigFile(VAL_STR (value)); }

```

Comment:
If possible, I'd prefer this `if-elseif...` cascade to be excluded from the auto-formatting and left as-is, perhaps just aligned to columns.